src/options.yml

```diff
@@ -225,6 +225,8 @@ INTERNAL: # Options for debugging the TruffleRuby implementation
     # Instrumentation to debug performance
     REGEXP_INSTRUMENT_CREATION: [regexp-instrument-creation, boolean, false, Enable instrumentation to gather stats on regexp creation]
     REGEXP_INSTRUMENT_MATCH: [regexp-instrument-match, boolean, false, Enable instrumentation to gather stats on regexp matching]
+    THREAD_WARN: [thread-warn, boolean, false, Warn with a backtrace when new threads are created]
+    FIBER_WARN: [fiber-warn, boolean, false, Warn with a backtrace when new fibers are created]
```

Comment:
They are not really warnings (no `warning:` prefix, not calling `Warning.warn`), they just print something like this:
```
fiber:-e:1:in `__allocate__'
	from -e:1:in `<main>'
or
thread:<internal:core> core/thread.rb:109:in `new'
	from -e:1:in `<main>'
```

Could you rename these options to `--backtraces-thread` / `--backtraces-fiber` or maybe `--backtraces-new-thread` / `--backtraces-new-fiber`? And the description should then just be `Show a backtrace when new threads/fibers are created`

Reply:
Done - and I added a new section for `# Print backtraces at key events` as most of these aren't really about backtraces.